lucien-ios/AddNewComicBook/AddComicViewController.swift

```diff
@@ -136,13 +128,22 @@ final class AddComicViewController: UIViewController {
         releaseDateToolBar.sizeToFit()
         releaseDateTextField.inputAccessoryView = releaseDateToolBar
 
-        // Genre
         configurePickerUIButton(button: selectAGenreButton)
-
-        // Condition
         configurePickerUIButton(button: selectAConditionButton)
     }
 
+    private func setupCoverPhotoButton() {
+        // The attributed title for the coverPhotoButton is set on the storyboard, but if an attributed title is created for the highlighted state
```

Comment:
I'd double check that this can't all be done in the xib/storyboard. Probably don't need attributed titles either, can just set the font and color separately.

Reply:
Nevermind, that centeredAligned text is the issue, so you probably need the attributed strings. I'd just set it in the xib if you can though.